site/en/blog/renderingng-architecture/index.md

```diff
@@ -0,0 +1,502 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Overview of the RenderingNG architecture'
+description: >
+  This post explains the components of the RenderingNG architecture,
+  and how the rendering pipeline flows through them.
+date: 2021-07-20
+authors:
+  - chrishtr
+---
+
+
+In a [previous post](/blog/renderingng/),
+I gave an overview of the RenderingNG architecture goals and key properties.
+This post will explain how its component pieces are set up,
+and how the rendering pipeline flows through them.
+
+Starting at the highest level and drilling down from there,
+the tasks of rendering are:
+
+1. Render contents into pixels on the screen.
+1. Animate visual effects on the contents from one state to another.
+1. Scroll in response to input.
+1. Route input efficiently to the right places so that developer scripts and other subsystems can respond.
+
+The contents to render are a stream of raw input events from touch screens,
```

Comment:
I was trying to avoid the nested lists, which didn't really seem required here. The next paragraph then moved onto talking about frames, which seemed a more natural follow-on from "Each frame includes:" though I can swap them around, I think it would be good to avoid the nested list there.

Reply:
Avoiding the nested list seems fine, but I think the order needs to be swapped.